### PR TITLE
component-icon, background-image mode

### PR DESCRIPTION
**Pull request by fabiosantoscode**
So, this adds background-image CSS classes `.icon .icon--ICONNAME` for each icon type.

Adding a new icon is just a matter of adding it to the SVG folder and running the build script.

### CODE REVIEW

File: example.es6 [hunk 1]
```diff
@@ -25,26 +25,133 @@ export default (
...
+    />
+    <Icon icon="economist" size="200px"
+      background="white"
+    />
+    <h1>Icons in background-image mode (IE: no extra svg download)</h1>
+    <p>
+      To use this you need to add <code>@import "@economist/component-icon/backgrounds/ICONNAME.css";</code>
+      to your postCSS.
+    </p>
+    <span className=".icon .icon--facebook">
```
Comment (umbobabo) [inline example.es6]: The output generated in the example for each of this icons is not correct:
- Wrong
<span class=".icon .icon--facebook" data-reactid=".0.19">Text for a11y and SEO purposes.</span>
- Correct
<span class="icon icon--facebook" data-reactid=".0.19">Text for a11y and SEO purposes.</span>

Please correct.

File: example.css [hunk 2]
```diff
@@ -2,3 +2,25 @@
   border-radius: 50%;
   padding: 20px;
 }
+
```
Comment (umbobabo) [inline example.css]: In this file, for example purpose you should provide a fixed width and height for the icons to modify the span to better understanding of the functionality.

**Comment by umbobabo** [comment]
Please review the notes.
When you open a PR please report eventually steps required that the reviewer developer may be not be aware of like 'Run npm build to obtains the required assets to test the component'.

File: example.es6 (hunk 1 above)
Comment (fabiosantoscode) [inline example.es6]: Not generated output, but it's fixed now!

**Comment by fabiosantoscode** [comment]
@umbobabo I've fixed what you said in your notes

**Comment by fabiosantoscode** [comment]
...And I've added my build script to npm build, and added `background-repeat: no-repeat`.

**Comment by keithamus** [comment]
backgrounds/* and bundle.css should have been added to `.gitignore`

**Pull request merged**